CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 
 * Added support to configure the default *.sln file loaded when opening a project with multiple *.sln files in the root. _(Contributed by [@janaka](https://github.com/janaka))_ (PR: [#2053](https://github.com/OmniSharp/omnisharp-vscode/pull/2053)) 
 
+* Added support for tracking opening, closing and changing of virtual documents that don't exist on disk(PR: [#2436] (https://github.com/OmniSharp/omnisharp-vscode/pull/2436)) _(Contributed by [@NTaylorMullen](https://github.com/NTaylorMullen))
```

Comment:
add a period after `disk` and a space before `(PR:`. Should be `disk. (PR:`

Reply:
Also, remove the space between `[#2436]` and `(https://`. Otherwise, your markdown link is broken.